### PR TITLE
Allow custom gomplate image and custom temps directory

### PR DESCRIPTION
**Pull request by HotThoughts**
Resolves
- #5
- #8

### CODE REVIEW

**Comment by HotThoughts** [comment]
@tczekajlo This PR is ready for review. Please 👀 if you have time.

File: action.yml [hunk 1]
```diff
@@ -153,7 +161,7 @@ runs:
         fi
 
         # Set docker args
-        echo DOCKER_ARGS="--rm --tmpfs /.config -e MPLCONFIGDIR=/tmp/.mplconfigdir \
+        echo DOCKER_ARGS="--rm --tmpfs ${{ inputs.tmpfs_directory }} -e MPLCONFIGDIR=/tmp/.mplconfigdir \
```
Comment (tczekajlo) [inline action.yml:164]: I'd add `-e HOME=/app` as well.

**Review by tczekajlo — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (HotThoughts) [reply action.yml]: Added 👍🏻

**Review by tczekajlo — APPROVED** [review]
LGTM 🎖️

**Pull request merged**